yarn.lock

```diff
@@ -7140,25 +7140,26 @@ fsevents@^1.2.7:
     bindings: ^1.5.0
     nan: ^2.12.1
   checksum: ae855aa737aaa2f9167e9f70417cf6e45a5cd11918e1fee9923709a0149be52416d765433b4aeff56c789b1152e718cd1b13ddec6043b78cdda68260d86383c1
+  conditions: os=darwin
```

Comment:
_this_ is really interesting - not sure exactly what it means when we try to build on a non-macOS system.